buildingmotif/dataclasses/model.py

```diff
@@ -147,6 +147,7 @@ def validate(
         self,
         shape_collections: Optional[List[ShapeCollection]] = None,
         error_on_missing_imports: bool = True,
+        shacl_engine: str = "default",
```

Comment:
would it be easier if the default value was `None` and the engine was only resolved at the level at which it was directly used? No need to check if it's default and resolve the value at a high level if at the layer where it is actually used that resolution would happen anyways.

Reply:
I agree. I just pushed the change